sdk-lib/src/androidTest/java/cash/z/ecc/android/sdk/db/DatabaseCoordinatorTest.kt

```diff
@@ -124,28 +125,28 @@ class DatabaseCoordinatorTest {
             DatabaseNameFixture.newDbWal(name = DatabaseCoordinator.DB_DATA_NAME)
         )
 
-        assertTrue(originalDbFile.existsSuspend())
-        assertTrue(originalDbJournalFile.existsSuspend())
-        assertTrue(originalDbWalFile.existsSuspend())
+        assertTrue(originalDbFile.exists())
```

Comment:
In this case, the suspending functions don't give us really any benefit since it is a test method.  I. changed them due to method visibility changes for the File extensions during an intermediate step of the refactoring.